hermes/bin/src/wasm/module.rs

```diff
@@ -110,6 +110,109 @@ impl Module {
     }
 }
 
+#[cfg(feature = "bench")]
+#[allow(dead_code)]
+pub mod bench {
+    use super::*;
+
+    /// Benchmark for executing the `init` event of the Hermes dummy component.
+    /// It aims to measure the overhead of the WASM module and WASM state initialization
+    /// process.
+    pub fn module_hermes_component_bench(b: &mut criterion::Bencher) {
+        struct Event;
+        impl HermesEventPayload for Event {
+            fn event_name(&self) -> &str {
+                "init"
+            }
+
+            fn execute(&self, instance: &mut ModuleInstance) -> anyhow::Result<()> {
+                instance
+                    ._instance
+                    .hermes_init_event()
+                    .call_init(&mut instance._store)?;
+                Ok(())
+            }
+        }
+
+        let mut module = Module::new(
+            "app".to_string(),
+            include_bytes!("../../benches/component.wasm"),
+        )
+        .unwrap();
+
+        b.iter(|| {
+            module.execute_event(&Event).unwrap();
+        });
+    }
+
+    /// Benchmark for executing the `foo` WASM function of the tiny component.
+    /// The general flow of how WASM module is instantiated and executed is the same as in
+    /// the previous one `module_hermes_component_bench`.
+    /// It aims to compare how the size of the component affects on the execution time.
+    pub fn module_small_component_bench(b: &mut criterion::Bencher) {
+        let wat = r#"
+            (component
+                (core module $Module
+                    (export "foo" (func $foo))
+                    (func $foo (result i32)
+                        i32.const 1
+                    )
+                )
+                (core instance $module (instantiate (module $Module)))
+                (func $foo (result s32) (canon lift (core func $module "foo")))
+                (export "foo" (func $foo))
+            )"#;
+
+        let engine = Engine::new().unwrap();
```

Comment:
Just curious, why we can use `unwrap`

Reply:
Just because it is inside benchmarks :) `cargo clippy` does not applied to them.
If we want we need to modify catalyst-ci as well, but it is a good point, worth to raise on the tech discussion.